### PR TITLE
WD-3904-credentials-schedule

### PR DESCRIPTION
**Pull request by abhigyanghosh30**
## Done

- [List of work items including drive-bys - remember to add the why and what of this work.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit /credentials/schedule?contractItemID=<contractItemID>

## Issue / Card

Fixes # [WD-3904](https://warthogs.atlassian.net/browse/WD-3904)

## Screenshots

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-3904]: https://warthogs.atlassian.net/browse/WD-3904?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by camillet16** [comment]
@abhigyanghosh30 where can I view the demo, demohaus only leads to ubuntu.com when I add the '/credentials/schedule?contractItemID=' it's a 404.

**Comment by abhigyanghosh30** [comment]
> @abhigyanghosh30 where can I view the demo, demohaus only leads to ubuntu.com when I add the '/credentials/schedule?contractItemID=' it's a 404.

@camillet16  [Try this link](https://ubuntu-com-12988.demos.haus/credentials/schedule?contractItemID=210)

File: templates/credentials/schedule.html [hunk 1]
```diff
@@ -24,31 +21,34 @@ <h5 class="p-notification__title">Error</h5>
   </div>
   {% endif %}
   <div class="row">
+    <div class="col-6">
+      <h1 style="font-weight: 500">Schedule your exam</h1>
```
Comment (camillet16) [inline templates/credentials/schedule.html:25]:
```suggestion
      <h2><strong>Schedule your exam</strong></h2>
      //new branding is currently being updated on canonical.com. This is the pattern they're using at the moment.
```

File: templates/credentials/schedule.html [hunk 2]
```diff
@@ -24,31 +21,34 @@ <h5 class="p-notification__title">Error</h5>
...
+        <label class="p-heading--5" for="exam-name">Select your exam</label>
         <select id="exam-name" name="name" required>
           <option value="linux-essentials">CUE: Linux</option>
         </select>
-        <label class="p-heading--4" for="exam-timezone">Select your timezone</label>
+        <label class="p-heading--5" for="exam-timezone">Select your timezone</label>
         <select id="exam-timezone" name="timezone" required>
         </select>
-        <label class="p-heading--4" for="exam-date">Select your preferred day</label>
+        <label class="p-heading--5" for="exam-date">Select your preferred day</label>
```
Comment (camillet16) [inline templates/credentials/schedule.html:38]:
```suggestion
        <label class="p-heading--5 u-no-margin" for="exam-date">Select your preferred day</label>
```

File: templates/credentials/schedule.html [hunk 3]
```diff
@@ -24,31 +21,34 @@ <h5 class="p-notification__title">Error</h5>
...
         <p>You can schedule or reschedule your exam up to 24 hours in advance.</p>
         <input type="date" id="exam-date" name="date" value="{{date}}" min="{{min_date}}" max="{{max_date}}" required />
-        <label class="p-heading--4" for="exam-time">Select your preferred time</label>
+        <label class="p-heading--5" for="exam-time">Select your preferred time</label>
```
Comment (camillet16) [inline templates/credentials/schedule.html:41]:
```suggestion
        <label class="p-heading--5 u-no-margin" for="exam-time">Select your preferred time</label>
```

**Comment by camillet16** [comment]
The background color is #f3f3f3 or $color-paper. thank you!

**Comment by abhigyanghosh30** [comment]
> The background color is #f3f3f3 or $color-paper. thank you!

This is for the entire page or the input boxes? Because currently, the input boxes have #f2f2f2 as the default colour.

**Comment by abhigyanghosh30** [comment]
> Screen Shot 2023-06-26 at 6 09 37 PM
> * margin-bottom: 1.5rem on top of the 1.2 rem of the `<select>`
> 
> This increases the separation between the selections and the next label. It would total 2.7rem.

Does this apply to all the input sections?

**Comment by camillet16** [comment]
> > The background color is #f3f3f3 or $color-paper. thank you!
> 
> This is for the entire page or the input boxes? Because currently, the input boxes have #f2f2f2 as the default colour.

The input box color with paper background is adjusted to rgba(0,0,0,.04)

**Comment by camillet16** [comment]
> Does the 2.7rem margin apply to all the input sections?

Yes 😄

File: templates/credentials/schedule.html [hunk 4]
```diff
@@ -24,31 +21,34 @@ <h5 class="p-notification__title">Error</h5>
   </div>
   {% endif %}
   <div class="row">
+    <div class="col-6">
+      <h2><strong>Schedule your exam</strong></h2>
```
Comment (mrgnr) [inline templates/credentials/schedule.html:25]: This should be `h1` to follow accessibility standards. You can add `class="p-heading--2"` to style it as an `h2`.

**Review by mrgnr — APPROVED** [review]

**Comment by abhigyanghosh30** [comment]
@camillet16 made changes as requested. Please review again

File: static/sass/styles.scss [hunk 5]
```diff
@@ -1369,6 +1369,9 @@ hr.p-separator.is-shallow {
   }
 }
 
+.is-paper--input {
+  background-color: rgba(0, 0, 0, 0.04);
+
```
Comment (albertkol) [inline static/sass/styles.scss:1374]: CSS is broken.

Comment (albertkol) [reply static/sass/styles.scss:1374]: It might fix all the failing pipelines and if design is happy then we can try an merge it today.

**Review by albertkol — CHANGES_REQUESTED** [review]

**Review by albertkol — APPROVED** [review]

**Pull request merged**